bin/meshroom_photogrammetry

```diff
@@ -125,6 +128,29 @@ if args.overrides:
             for attrName, value in overrides.items():
                 graph.findNode(nodeName).attribute(attrName).value = value
 
+if args.paramOverrides:
+    print("\n")
+    import re
+    reExtract = re.compile('(\w+)([:.])(\w+)=(.*)')
+    for p in args.paramOverrides:
+        result = reExtract.match(p)
+        if not result:
+            raise ValueError('Invalid param override: ' + str(p))
+        node, t, param, value = result.groups()
+        if t == ':':
+            nodesByType = graph.nodesByType(node)
+            if not nodesByType:
+                raise ValueError('No node with the type "{}" in the scene.'.format(node))
+            for n in nodesByType:
+                print('Overrides {node}.{param}={value}'.format(node=node, param=param, value=value))
+                n.attribute(param).value = value
+        elif t == '.':
+            print('Overrides {node}.{param}={value}'.format(node=node, param=param, value=value))
+            graph.findNode(node).attribute(param).value = value
```

Comment:
just asking, `findNode(node)` can fail in the same way as `nodesByType(node)`? Shall we manage the error?
Also, very minor thing, I'd put the print after the setting is done so it is not print in case of failure (hopefully `findNode(node)` or `attribute(param)` will print or raise some kind of error?)

In case, the same for the previous `nodesByType(node)`

Reply:
findNode is looking for one specific node and fails with an explicit message if it doesn't find it.
nodesByType looks for all nodes of a specific type and can return an empty list without error (that's why there is a specific check to provide an error message).
I put the print before so it's more visible in the error message :s, the last NODE.PARAM is printed and then you have the error message saying that it doesn't find PARAM for instance.